other/upgrade.php

```diff
@@ -3374,9 +3378,9 @@ function move_db_last_error_to_cachedir()
 	}
 }
 
-/******************************************************************************
-******************* Templates are below this point ****************************
-******************************************************************************/
+/* ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+                        Templates are below this point
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ */
```

Comment:
You should follow SMF's comment standard in here. While I understand the reasoning consistency should be fixed.

Reply:
If we are going by overall consistency the whole section is not consistent, and it is also non-existent in installer.